homeassistant/components/ssdp/__init__.py

```diff
@@ -351,6 +351,10 @@ async def _ssdp_listener_callback(
         ssdp_change = SSDP_SOURCE_SSDP_CHANGE_MAPPING[source]
         await _async_process_callbacks(callbacks, discovery_info, ssdp_change)
 
+        # Config flows should only be created for alive messages from alive devices
+        if ssdp_change != SsdpChange.ALIVE:
```

Comment:
An update message type would be a valid one to work with as well, especially since it allows config flows to update existing config entries. 

So maybe ignore only the `byebye` advertisement?

Reply:
I'm not so sure, based on what the [UPnP Device Architecture spec](http://upnp.org/specs/arch/UPnP-arch-DeviceArchitecture-v1.1.pdf) (section 1.2.4) says about what to do with ssdp:update messages. My interpretation is that the device will not give any new information in the update message, other than a replacement BOOTID, and that new values for LOCATION etc. will appear in subsequent ssdp:alive notifications and M-SEARCH responses. And based on that, only the `SsdpChange.ALIVE` case is useful for config flow.

Reply:
Related: https://github.com/home-assistant/core/pull/56591

Reply:
@chishm That message will still contain the host/port, which can be updated.

Reply:
The host, and port should remain the same as from previous ssdp advertisements with the same BOOTID, given they're sent via the same interfaces. However I see your point that it's not causing issues to allow the `ssdp:update` message to kick off a config flow, and will adjust this PR accordingly.